app/components/works/description_component.html.erb

```diff
@@ -60,6 +60,6 @@
 
   <%= render Works::RelatedWorkComponent.new(form: form) %>
 
-  <%= render RelatedLinkComponent.new(form: form) %>
+  <%= render RelatedLinkComponent.new(form: form, key: 'work.related_link') %>
```

Comment:
I'm not seeing that i18n key in `config/locales/en.yml`. Did I miss it?

Reply:
It's not there yet (different ticket), but this component requires a key.  It gracefully handles missing translations.

Reply:
OK.